baseline.gradle

```diff
@@ -73,39 +73,57 @@ subprojects {
       options.errorprone.errorproneArgs.addAll (
           // error-prone is slow, don't run on tests/generated-src/generated
           '-XepExcludedPaths:.*/(test|generated-src|generated)/.*',
-          // specific to Palantir
-          '-Xep:ConsistentLoggerName:OFF',  // Uses name `log` but we use name `LOG`
-          '-Xep:FinalClass:OFF',
-          '-Xep:PreferSafeLoggingPreconditions:OFF',
-          '-Xep:PreferSafeLoggableExceptions:OFF',
-          '-Xep:Slf4jLogsafeArgs:OFF',
-          '-Xep:RawTypes:OFF',
-          // enforce logging conventions
-          '-Xep:LoggerEnclosingClass:ERROR',
-          '-Xep:PreferStaticLoggers:ERROR',
-          '-Xep:Slf4jThrowable:ERROR',
+          '-Xep:AnnotateFormatMethod:ERROR',
```

Comment:
it would be good to structure this so that it's clearly visible what changed here. Atm it's difficult to tell what was added

Reply:
Agreed, it looks like a mess.
Does reviewing commit by commit help?
One of the early commits is `Sort error-prone configuration options`. Obviously it blurs the picture for this PR, but should help going forward. Do you think such a change helps?

Reply:
the change itself doesn't seem related to dropping JDK 8 support, so I don't see a strong reason to sort the configurations in this PR. Also most people (me included) typically look at the entire diff rather than looking at individual commits (as all commits will be squashed together anyway at the end)

Reply:
> as all commits will be squashed together anyway at the end

is it a project rule, or the merging person's preference?

i tend to keep the PR's commit tidy & in order

> the change itself doesn't seem related to dropping JDK 8 support, so I don't see a strong reason to sort the configurations in this PR.

you're right!
There is "drop 8" change, which doesn't affect error-prone by itself, and then there is also update error-prone plugin commit which does.
I keep these changes in one PR because it's easier to manage this way & we're not ready to drop 8 yet. I can separate when needed. For example, do you think it'd help if i extracted `Sort error-prone configuration options` to a separate PR?

Reply:
Squashing commits is a project rule. I think it's probably best to extract the error-prone stuff into a separate PR as that will reduce noise on this PR

Reply:
I extracted https://github.com/apache/iceberg/pull/10540. Thanks for the tip!